packages/components/src/components/vertical-tab/vertical-tab-header-title.jsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
+import Text from '../text';
 
 const VerticalTabHeaderTitle = ({ header_title }) => (
     <div className='dc-vertical-tab__header-title'>
-        <p>{header_title}</p>
+        <Text as='p' size='s' color='prominent' align='center' weight='bold'>
```

Comment:
We can skip `'s'` for **size** as it's the default value.

Reply:
As Aaron requested for change,I made font size and line height required in this PR: https://github.com/binary-com/deriv-app/pull/2030
I am not sure why they are not required anymore!

Reply:
They're not required in the latest `master`. Correct me if I'm mistaken.
So if they're not required and have default values, we better don't pass them down.

Reply:
Yes, they are not but I don't know how they have been removed!! I will remove these props for now, and make them required again in another PR. Thanks for your attention.